cmd/template/framework/files/makefile.tmpl

```diff
@@ -5,7 +5,7 @@ all: build
 
 build:
 	@echo "Building..."
-	@go build -o main cmd/api/main.go
+	@go build -o tmp/main cmd/api/main.go
```

Comment:
i think mostly people expect  build file to be in root of the folder so we should keep it that way